packages/@react-spectrum/table/src/table.css

```diff
@@ -45,5 +45,6 @@
 }
 
 .react-spectrum-Table-cellContents--hideHeader {
-  text-overflow: clip;
+  margin: 0;
+  padding: 0;
```

Comment:
overrides this https://github.com/adobe/react-spectrum/blob/main/packages/@adobe/spectrum-css-temp/components/table/index.css#L159-L160 as it increases the width of the cell and we don't need it for this action column

Reply:
Lets move this class into spectrum-css as the hideHeader is a spectrum requirement
That way we can make this association a lot clearer
Please move it to be right after the lines you linked in your comment